nuxeo-features/nuxeo-platform-oauth/src/main/java/org/nuxeo/ecm/platform/oauth2/request/AuthorizationRequest.java

```diff
@@ -70,6 +87,46 @@
 
     public static final String STATE = "state";
 
+    public static AuthorizationRequest from(HttpServletRequest request) {
+        return new AuthorizationRequest(request);
+    }
+
+    /**
+     * @since 8.10-HF34
```

Comment:
Not sure all the `@since` are needed given that this is also in master. Or maybe `@since 9.2`?